src/SCRIPTS/BF/ui.lua

```diff
@@ -85,6 +90,7 @@ local function createPopupMenu()
         { t = "save page", f = saveSettings },
         { t = "reload", f = invalidatePages },
         { t = "reboot", f = rebootFc },
+        { t = "acc cal", f = accCal },
```

Comment:
Not sure if we want to make this a bit more verbose, like pop up a dialog saying something like 'Make sure the craft is level and stable, then press [Enter] to continue, or [Cancel]'?
Otherwise I suspect unsuspecting newbie pilots will 'test' this function and then end up un-calibrating their craft when they 'try' this.

Reply:
Good point! I'll see what I can do. I think the best approach would be to just make it a special page that can only be entered from the popup menu. I looked into the built in popup dialogue in opentx but it's very limited and behaves differently on different radios

Reply:
@mikeller It's done. I updated the first post with a description of how this works with pictures